integration/benchmark/account/account.go

```diff
@@ -12,62 +12,70 @@ import (
 )
 
 type FlowAccount struct {
-	Address *flowsdk.Address
-	ID      int
-
-	keys *keystore
+	Address    flowsdk.Address
+	keys       *keystore
+	PrivateKey crypto.PrivateKey
+	HashAlgo   crypto.HashAlgorithm
```

Comment:
I think the crypto has a data structure that holds all of these values (hash, private key, ...)

Reply:
I didn't find anything. There is the AccountKey, but that has extra data.